tests/test_revision.py

```diff
@@ -28,27 +29,49 @@ def test_revision_init():
     assert test_revision.user == "Ss112"
     assert test_revision.userid == 1286970
 
+
+def test_contains_tag():
+    '''tests contain tag function inside revision class'''
+    test_revision = Revision({})
+    test_revision.tags = ["Reverted", "wikieditor"]
+    assert test_revision.contains_tag(["Reverted"]) is True
+    test_revision.tags = []
+    assert test_revision.contains_tag(["Reverted"]) is False
+    assert test_revision.contains_tag([""]) is False
+    test_revision.tags.append("wikieditor")
+    assert test_revision.contains_tag(["wikieditor"]) is True
+    assert test_revision.contains_tag(["wikieditor", "somethingelse"]) is False
+    test_revision.tags.append("Techno")
+    test_revision.tags.append("somethingelse")
+    assert test_revision.contains_tag(["wikieditor", "somethingelse"]) is True
+    test_revision.tags = []
+    assert test_revision.contains_tag(["wikieditor", "somethingelse"]) is False
+
+
 def test_get_content():
     """ Tests get_content method against known correct output """
     test_revision = Revision({})
     test_revision.revid = 1127195995
     content = test_revision.get_content()
-    content = content[0:content.index('<!')] # remove variable mwparser cmt
+    content = content[0:content.index('<!')]  # remove variable mwparser cmt
     with open('tests/resources/revision-get_contents.html', 'r', encoding='utf-8') as in_file:
         f_content = "".join(in_file.readlines())
         f_content = f_content.replace("\\\\", "\\")
         assert f_content == content
 
+
 def test_get_diff():
     """Tests get_diff method against known correct output"""
     test_revision = Revision({})
     test_revision.revid = 1127195995
-    test_revision.parentid =1126322774
```

Comment:
I'm noticing a lot of stray spaces and newlines added/removed, any reason for this? Seems like we should be careful not to mess with the formatting too much, it is bad style to be inconsistent about spacing of comments, number of lines between function defs, etc.